runtime/dart_vm_lifecycle.cc

```diff
@@ -88,7 +88,7 @@ DartVMRef DartVMRef::Create(Settings settings,
   gVM = vm;
 
   if (settings.leak_vm) {
```

Comment:
Nit: can we have a comment in settings.h that explains why leak_vm is default to true (I guess that it's to preserve our old behavior)? The name `leak_vm` sounds scary to me, especially considering that its default value is true...

Reply:
Missed this. Sure will update the comment in an incoming patch. You're right, it is to preserve old behavior. Applications tend to "warm-up" the VM and depend on the VM running when launching subsequent apps. Not leaking by default breaks these workarounds.